### PR TITLE
fix(docs): correct the tutorial: inputs, outputs and triggers docs

### PR DESCRIPTION
**Pull request by shrutimantri**
Corrected the tutorial: inputs, outputs and triggers docs

### CODE REVIEW

File: content/docs/02.tutorial/02.inputs.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Inputs can be accessed in any task using a special `{{ inputs.input_name }}` [va
 
 ## Defining inputs
 
-Similarly to tasks, inputs is a list of key-value pairs. Each input must have a `name` and a `type`. You can also set `defaults` for each input. Setting default values for an input is always recommended, especially if you want to run your flow on a schedule.
+Similar to task, input is a list of key-value pairs. Each input must have a `name` and a `type`. You can also set `defaults` for each input. Setting default values for an input is always recommended, especially if you want to run your flow on a schedule.
```
Comment (Skraye) [inline content/docs/02.tutorial/02.inputs.md:15]: need to keep `tasks` and `inputs` as we mention the property

**Review by Skraye — CHANGES_REQUESTED** [review]
Really small changes

File: content/docs/02.tutorial/02.inputs.md (hunk 1 above)
Comment (shrutimantri) [reply content/docs/02.tutorial/02.inputs.md:15]: Makes sense.

**Review by Skraye — APPROVED** [review]
LGTM

**Pull request merged**